docs/asciidoc/static/advanced-pipeline.asciidoc

```diff
@@ -116,6 +116,14 @@ filter {
 }
 --------------------------------------------------------------------------------
 
+To set the stdout output to json format:
+[source,json]
+--------------------------------------------------------------------------------
+output {
+    stdout {codec=>rubydebug} 
```

Comment:
Well, both work but the preceding paragraph says "To set the stdout output to json format" and then you can't set the output codec to rubydebug because that's not JSON.